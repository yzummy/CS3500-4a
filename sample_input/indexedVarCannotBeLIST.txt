{
    A = list(1, 2, 3);
    A[[1]] = list("hello", "goodbye");
    print("the end")
}
