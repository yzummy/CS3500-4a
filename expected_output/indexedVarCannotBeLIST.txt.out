Line 3: Arg 1 cannot be list
